integration-tests/README.md

```diff
@@ -31,9 +31,24 @@ import { ALITH_ADDRESS } from "@moonwall/util";
 
 ### Usage Examples (non-exhaustive)
 
+You should have installed `python` for using `sqlite3` and then used `pnpm rebuild && pnpm rebuild sqlite3`.
+
+Run `pnpm install` in the `integration-tests` folder before running any of the following commands.
+
+Useful for integration testing:
+
+- `./integration-tests/scripts/deploy-zombienet.sh` - Deploy a local relay-parachain network for zombienet.
```

Comment:
What did you get?

Reply:
Ooops, screenshot didn't load
<img width="1920" alt="Screenshot 2024-01-21 at 19 30 06" src="https://github.com/zeitgeistpm/zeitgeist/assets/49552287/d04c0e26-f131-4ca2-8e4b-04062509ca4a">

Reply:
You have the same error like previously. It's because you are missing the polkadot binary. 

Can you run the following first?

```bash
cd integration-tests
./scripts/download-polkadot.sh
```

Reply:
Exactly what you can see there. It starts the network by spawning a zeitgeist node (alice) and two polkadot validators. You can do whatever you want on polkadot js now.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1194/commits/6e574ff3536bab823e47f4bebbb124232de521b2